Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -5325,6 +5325,60 @@ def to_context(self):
         return indicator_relationship_context
 
 
+# -------------------------------- Threat Intel Objects ----------------------------------- #
+
+class ThreatIntel:
+    """XSOAR Threat Intel Objects"""
+
+    class ObjectsNames(object):
```

Comment:
does it have dups with the FeedIndicatorType enum? in any case, put them next to(before/after) each other.

Reply:
done